eldev-util.el

```diff
@@ -1091,16 +1102,17 @@ Also, eat up several options from BODY if present:
                      (signal 'eldev-error (list "%s exited with error code %d" (eldev-message-upcase-first description) exit-code))))))
            ,@(or body '(exit-code)))))))
 
-(defun eldev--forward-process-output (&optional header-message header-if-empty-output only-when-verbose)
+(defun eldev--forward-process-output (&optional header-message header-if-empty-output only-when-verbose nolf)
```

Comment:
Actually, it should always work as if `nolf` was specified, even for existing invocations. Drop the parameter, just change the function. For `eldev-verbose` it should stay as it is, just in case the invoked process doesn't print ending newline. But for `eldev-output` we should just pass generated output without changes, not even supplying trailing `lf`.

Reply:
So just to double check, the function should look like this? :
```elisp
(defun eldev--forward-process-output (&optional header-message header-if-empty-output only-when-verbose)
  (if (= (point-min) (point-max))
      (when header-if-empty-output
        (eldev-verbose header-if-empty-output))
    (when header-message
      (eldev-verbose header-message))
    (if only-when-verbose
        (eldev-verbose "%s" (buffer-string))
      (eldev-output :nolf "%s" (buffer-string)))))
```
In which case, am I ok to fix the existing tests which expect a newline from this fn, e.g. `eldev-emacs-2`?

Reply:
Yes. Adjust all tests that depend on this accordingly, they are not meant to be set in stone.